ethereum/.env

```diff
@@ -0,0 +1,3 @@
+BRIDGE_DEPLOY_PATH=/work/mangata-ws/mangata-bridge/test
+ETH_ENDPOINT_WS=ws://ganache:8545/
+SUB_ENDPOINT_WS=ws://parachain:9944/
```

Comment:
`SUB_ENDPOINT` from relay environment have the same value (and should do it always as far as I can see). Can we combine these two variables into one?

Reply:
Sure.